admin_manual/configuration_server/config_sample_php_parameters.rst

```diff
@@ -1507,18 +1507,12 @@ This enables debug logs for the windows_network_drive app.
 Overriding Existing Parameter Values Using Environment Variables
 ----------------------------------------------------------------
 
-ownCloud supports overriding existing configuration values for the web UI,
-command line, and Cron environments, using `environment variables`_.
-Doing so avoids the need to storing credentials, and other *private* data,  data
-which you never want to leak out, in either code or source repositories. 
-What’s more, by taking this approach, code changes aren’t required to deploy to
-multiple environments.
-
-To override an existing setting, you need to export an environment variable,
-which has the same name as the one which you want to override, prefixed with
-``OC_``. 
-For example, if you wanted to override the value of ``dbname``, you would set
-the environment variable ``OC_dbname``.
+ownCloud supports the ability to override the *web UI*, *command line*, and *Cron* environments’ settings by using `environment variables`_.
+By doing so, you avoid the need to store credentials and other sensitive data (such as secret keys), which you never want to store in code, and which can potentially leak out. 
```

Comment:
Just end the Sentence after `sensitive data`. From a security perspective it doesn't make a difference for a hacker if the passwords are stored in a environment variable or inside the code. Whenever the hacker can already access the code, he will definitely have plenty of ways to read them.

This is not a security feature, it's a convenience feature.